### ISSUE TITLE
Unlike uBlock, uMatrix's logger defaults to "all" instead of the page where the logger was opened from

### ISSUE THREAD

**Issue by Rukachan** [gorhill/uMatrix#539]
Making it difficult to select the correct site when many tabs are open, especially if they have similar urls/titles.

**Comment by Rimmer7** [comment]
The uMatrix logger overall is less useful and flexible than the uBlock logger.

**Comment by gorhill** [comment]
Yes, it needs work to bring it on par with uBO's one. Another thing missing is the the rule which caused a request to be blocked/allowed, that would be useful too.
